lib/PrivilegedDonorContacts/hooks/useCategories/useCategories.js

```diff
@@ -0,0 +1,41 @@
+import { useQuery } from 'react-query';
+
+import {
+  useNamespace,
+  useOkapiKy,
+} from '@folio/stripes/core';
+
+import { LIMIT_MAX } from '../../../constants';
+import { CATEGORIES_API } from '../constants';
+import { useTranslatedCategories } from '../useTranslatedCategories';
+
+const DEFAULT_DATA = [];
+
+export const useCategories = (options = {}) => {
```

Comment:
`useCategories` and `useTranslatedCategories` are better to keep in `lib/hooks` because those are just a duplication of the same hooks in  `ui-organizations`  and `ui-plugin-find-contact` and if possible they should be reused from acq lib:
- https://github.com/folio-org/ui-organizations/blob/d0fdfefd1ff78304315979a4fa1593667cc2fa2f/src/common/hooks/useCategories/useCategories.js#L14
- https://github.com/folio-org/ui-plugin-find-contact/blob/3db47753ed6465a2c0538cffeed3daee5f902f52/findContact/hooks/useCategories/useCategories.js#L11